client/src/globals.ts

```diff
@@ -9,6 +9,9 @@ export const overflowCategoryLabel = ": all other labels";
 /* default "unassigned" value for user-created categorical metadata */
 export const unassignedCategoryLabel = "unassigned";
 
+/** Maximum number of cells a dataset can have in order to be included for display. */
+export const DATASET_MAX_CELL_COUNT = 200000;
```

Comment:
Two million is the max 🙆

```suggestion
export const DATASET_MAX_CELL_COUNT = 2_000_000;
```